benchmarks/src/jmh/java/io/grpc/DecompressorRegistryBenchmark.java

```diff
@@ -52,6 +52,7 @@
  */
 @State(Scope.Benchmark)
 public class DecompressorRegistryBenchmark {
+  private static final Joiner ACCEPT_ENCODING_JOINER = Joiner.on(',');
```

Comment:
This benchmark was moved to have the same package structure as the code being benchmarked so it was easier to expose visibility.

Maybe make AEJ package visibility below and just reference it here?